tests/Doctrine/Tests/TestUtil.php

```diff
@@ -61,16 +61,28 @@ private static function hasRequiredConnectionParams()
             $GLOBALS['db_type'],
             $GLOBALS['db_username'],
             $GLOBALS['db_password'],
-            $GLOBALS['db_host'],
-            $GLOBALS['db_name'],
-            $GLOBALS['db_port']
+            $GLOBALS['db_name']
+        )
+        && (
+            isset(
+                $GLOBALS['db_host'],
+                $GLOBALS['db_port']
+            ) || isset(
+                $GLOBALS['db_server']
```

Comment:
@deeky666 Sorry, for the delay. The sybase_ct `sybase_connect` function only accepts a server name. The further information to connect is then read from a file called "interfaces" where you can also specify information for connection encryption etc.
This is the standard behavior of working with Sybase ASE and Sybase IQ servers (when using the Sybase CT library provided by the database vendor)